spec.bs

```diff
@@ -1490,22 +1499,30 @@ Several APIs specific to fenced frames are defined on the {{Fence}} interface.
   };
 
   dictionary FenceEvent {
-    // This dictionary has two mutually exclusive modes that aren't represented as
-    // distinct IDL types due to distinguishability issues:
-    //
-    // When reporting to a preregistered destination (specified by enum), the following
-    // properties are used:
+    // This dictionary has two mutually exclusive modes:
+
```

Comment:
Why remove the line about distinguishability? These kinds of changes can come in another PR I think.